tests/source/macros.rs

```diff
@@ -218,11 +218,65 @@ make_test!(str_searcher_ascii_haystack, "bb", "abbcbbd", [
 }
 
 fn special_case_macros() {
-    // format!
-    let s = format!("Arr! While plunderin' the hold, we got '{}' when given '{}' (we expected '{}')", result, input, expected);
+    let q = eprint!("{}", 1);
+    let r = eprint!("{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+    let s = eprint!("{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26);
 
-    // assert!
```

Comment:
I dropped the comments that indicate which macro is being tested because after adding so much duplication I feel the macro names jump out really clearly. I can add comments back if anyone feels strongly about it.